vehicle1_commander:  # Name of your Node
  ros__parameters:
    init_pos_x: 86.395
    init_pos_y: -15.0
    init_pos_z: 0.0
    init_rot_x: 0.0
    init_rot_y: 0.0
    init_rot_z: -0.7072
    init_rot_w: 0.7072
    goal_pos_x: 86.395
    goal_pos_y: -300.0
    goal_pos_z: 0.0
    goal_rot_x: 0.0
    goal_rot_y: 0.0
    goal_rot_z: -0.7072
    goal_rot_w: 0.7072

vehicle2_commander:  # Name of your Node
  ros__parameters:
    init_pos_x: 86.395
    init_pos_y: -40.0
    init_pos_z: 0.0
    init_rot_x: 0.0
    init_rot_y: 0.0
    init_rot_z: -0.7072
    init_rot_w: 0.7072
    goal_pos_x: 86.395
    goal_pos_y: -280.0
    goal_pos_z: 0.0
    goal_rot_x: 0.0
    goal_rot_y: 0.0
    goal_rot_z: -0.7072
    goal_rot_w: 0.7072